--- config.def.h	2019-05-24 17:39:12.430554084 +0200
+++ dwm-patched/config.def.h	2019-05-24 17:17:23.295215462 +0200
@@ -1,5 +1,3 @@
-/* See LICENSE file for copyright and license details. */
-
 /* appearance */
 static const char *fonts[] = {
 	"monospace:size=10"
@@ -25,8 +23,7 @@
 	 *	WM_NAME(STRING) = title
 	 */
 	/* class      instance    title       tags mask     isfloating   monitor */
-	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
-	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
+  { "Firefox",  NULL,       NULL,       1,            0,           -1 },
 };
 
 /* layout(s) */
@@ -42,7 +39,7 @@
 };
 
 /* key definitions */
-#define MODKEY Mod1Mask
+#define MODKEY Mod4Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
 	{ MODKEY|ControlMask,           KEY,      toggleview,     {.ui = 1 << TAG} }, \
@@ -56,6 +53,10 @@
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
 static const char *termcmd[]  = { "st", NULL };
+static const char *suspendcmd[]  = { "susplock", NULL};
+static const char *raisevol[]  = { "pactl", "set-sink-volume", "0", "+8%", NULL};
+static const char *lowervol[]  = { "pactl", "set-sink-volume", "0", "-8%", NULL};
+static const char *mutevol[]  = { "pactl", "set-sink-volume", "0", "0%", NULL};
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
@@ -91,7 +92,11 @@
 	TAGKEYS(                        XK_7,                      6)
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
-	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ MODKEY|ShiftMask,             XK_Delete,      quit,      {0} },
+	{ 0,                            0x1008FF41 /*XF86Launch1*/, spawn,     {.v = suspendcmd} },
+	{ 0,                            0x1008FF13 /*XF86XK_AudioRaiseVolume*/, spawn,     {.v = raisevol} },
+	{ 0,                            0x1008FF11 /*XF86XK_AudioLoverVolume*/, spawn,     {.v = lowervol} },
+	{ 0,                            0x1008FF12 /*XF86XK_AudioMuteVolume*/, spawn,     {.v = mutevol} },
 };
 
 /* button definitions */
